website_odoo_debranding/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 {
     "name": "Remove Odoo Branding from Website",
-    "version": "15.0.1.0.0",
+    "version": "15.0.1.0.1",
```

Comment:
No need to change manifest version. It's done by the bot. Please revert this change.

Reply:
Sorry, I was under the impression the bot only updates the overview table in the README file, thanks for the feedback.

Reply:
Note that on my direct commit, as no bot was involved, I changed the manifest version, but that's only because of that. On normal PR operation, the bot is triggered.